src/atoms/Loader/Atom/loader-style.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../styles/resources.scss';
+@import '../../../styles/variables/variables';
```

Comment:
Instead of those two imports, you could simply do:  `@import '~styles/resources.scss'; `since everything is included there.


Also, please follow the existing convention about naming the style files. This should be named just `styles.scss`